modules/admin-ui-picard/app/src/utils/resourceUtils.js

```diff
@@ -98,6 +105,50 @@ export const prepareMetadataFieldsForPost = (metadataInfo, values) => {
     return metadataFields;
 }
 
+export const prepareSeriesMetadataFieldsForPost = (metadataInfo, values) => {
+    let metadataFields = [];
+
+    // fill metadataField with field information send by server previously and values provided by user
+    // Todo: What is hashkey?
+    for (let i = 0; metadataInfo.length > i; i++) {
+        let fieldValue = {
+            readOnly: metadataInfo[i].readOnly,
+            id: metadataInfo[i].id,
+            label: metadataInfo[i].label,
+            type: metadataInfo[i].type,
+            value: values[metadataInfo[i].id],
+            tabindex: i + 1,
+            $$hashKey: "object:123"
+        };
+        if (!!metadataInfo[i].translatable) {
+            fieldValue = {
+                ...fieldValue,
+                translatable: metadataInfo[i].translatable
+            };
+        }
+        if (!!metadataInfo[i].collection) {
+            fieldValue = {
+                ...fieldValue,
+                collection: [],
```

Comment:
is this collection supposed to be an empty array?

Reply:
yes, it's supposed to be empty. The endpoint wants a collection field if the metadata has a collection for value options (see the call in old UI).